bin/release.sh

```diff
@@ -7,7 +7,9 @@ USAGE:
 FLAGS:
     -h, --help      Show this help text and exit.
     -v, --verbose   Enable verbose output.
-    -d, --dry-run   Do not change any files or commit a git tag."
+    -d, --dry-run   Do not change any files or commit a git tag.
+    --publish-dry-run
+                    Perform a dry run on the publish step only."
```

Comment:
very tiny style nit: i would prefer to format this so that it's on the same line, and just go back and indent the other flags slightly more:
```suggestion
    --publish-dry-run    Perform a dry run on the publish step only."
```

Reply:
Done.